mantle/auth/google.go

```diff
@@ -26,15 +26,15 @@ import (
 	"golang.org/x/oauth2/google"
 )
 
-const GCEConfigPath = ".config/gce.json"
+const GCPConfigPath = ".config/gcp.json"
```

Comment:
I wonder though if this is going to break the pipeline.  It's probably better to look in *both* paths for now.

Reply:
The PR as a whole will probably break things.  For example, the pipeline [uses command-line options](https://github.com/coreos/fedora-coreos-pipeline/blob/main/jobs/kola-gcp.Jenkinsfile#L78-L80).  In this case I'd suggest that we just go around and fix them rather than trying to carry compat code.

Reply:
Hmm.  That will require a "flag" day in both our pipelines though, which has historically been quite painful.

Reply:
Maybe I can add WIP PR for the repos, which might be broken by this PR.

Reply:
One thing you can do now though is split the PR in two: 

- Cosmetic/doc changes
- Changes which will require pipeline changes

The first one can go in easily now.